interface.py

```diff
@@ -0,0 +1,25 @@
+class interface():
```

Comment:
Style recommendation: consider using `UpperCamelCase` for class names.

`main.py` uses `Interface()` (which it also needs to import).